UltraMapper.Tests/UltraMapper.Tests.csproj

```diff
@@ -1,34 +1,20 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net472</TargetFrameworks>
```

Comment:
I don't think Test projects should generate NuGet packages, so the section was removed. They don't get deployed anyways. Let me know if this is a valid concern

Reply:
That property is needed to inform what framework to compile against.

Reply:
I meant the nuget package section. That property has been removed as it's a multi target